Source/Flow/Public/Nodes/FlowPin.h

```diff
@@ -210,48 +210,3 @@ struct FLOW_API FPinRecord
 	FORCEINLINE static FString DoubleDigit(const int32 Number);
 };
 #endif
-
-// It can represent any trait added on the specific node instance, i.e. breakpoint
-USTRUCT()
-struct FLOW_API FFlowPinTrait
```

Comment:
Moved FFlowPinTrait to UFlowDebuggerSubsystem as it is used only in Editor module and represents only debug feature